### PR TITLE
(Paint.NET) Moving to Core-Team Repository, along with AU update script

### PR DESCRIPTION
**Pull request by The-Running-Dev**

### CODE REVIEW

File: automatic/paint.net/paint.net.nuspec [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - Active Online Community
+    </description>
+    <summary>
+    Paint.NET is image and photo manipulation software designed to be used on computers that run Windows.
+    </summary>
+    <tags>paint.net photo editing paint admin</tags>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/paint.net</packageSourceUrl>
+    <owners>chocolatey,the-running-dev</owners>
+    <dependencies>
+      <dependency id="Chocolatey-Package.extension" />
```
Comment (AdmiringWorm) [inline automatic/paint.net/paint.net.nuspec]: as can be seen from the failing appveyor build, this package does not exist.

Comment (The-Running-Dev) [inline automatic/paint.net/paint.net.nuspec]: Ops, my mistake, I'll remove that.

**Comment by AdmiringWorm** [comment]
@The-Running-Dev could you update this PR to the latest master?
I've added a few changes to the appveyor script which hopefully allows the appveyor run to pass.

**Comment by The-Running-Dev** [comment]
I just rebased from master. We all good?

**Comment by AdmiringWorm** [comment]
> We all good?

Unfortunately not, if you look at the commits for this PR, you'll notice that there is a few commits that shouldn't be there.
Try doing the following:

1. First delete the master branch (`git branch -D master`)
2. Run `git branch upstream/master -b master` (assuming you have this repo registered as upstream, can be checked with `git remote -v`
3. `git checkout feat-Paint.NET`
4. `git rebase master`
5. `git push --force`

Alternatively you can do the following instead of step 1 and 2.
Check out the master branch, the run `git reset upstream/master --hard` (assuming you have this repo registered as upstream)

**Comment by The-Running-Dev** [comment]
I thought that's what I did, I just followed your instructions, I don't see any changes.

**Comment by AdmiringWorm** [comment]
> I just followed your instructions, I don't see any changes.

you shouldn't see any changes locally (unless you run `git log`), however it should update the commit history on this PR when you push it (which doesn't look like you've done yet)

**Comment by AdmiringWorm** [comment]
btw, the problem with the uninstall script seems to be that the `PSChildName` isn't available on powershell v2, hence why uninstall doesn't work.

**Comment by The-Running-Dev** [comment]
Done!

**Comment by AdmiringWorm** [comment]
@The-Running-Dev try again, the uninstall script was re-added in the last merge commit you did.

**Comment by The-Running-Dev** [comment]
Jesus...

**Comment by AdmiringWorm** [comment]
perfect, now when you've opened a PR to Robs package repo removing the paint.net package, and it have been merged we can merge this in.
Please comment on this PR when it have been removed from Robs repo.

**Comment by The-Running-Dev** [comment]
I already submitted the PR to remove it from Rob's repo, and her merged it.

**Comment by AdmiringWorm** [comment]
I see, I didn't notice that.
In the future it would be nice if you include the link to the opened PR removing the package from the original package repository (just include it in the PR description)

**Pull request merged**